src/library/simplified/extension/marketingABTests.js

```diff
@@ -1,12 +1,38 @@
-var abTests = utag_data.context.tests;
-var marketingABTests = "";
-
-if (typeof abTests !== "undefined") {
-    for (var abTest=0; abTest<abTests.length; abTest++) {
-            var experimentId = utag_data.context.tests[abTest].experimentId;
-            var bucketValue = utag_data.context.tests[abTest].bucketValue;
-            marketingABTests += experimentId + "." + bucketValue + "|";
-    }
+var ABTestExpVals='';
+var tpid = utag_data['context.site.tpid'];
+var eapid = utag_data['context.site.eapid'];
+var guid = utag_data['guid'];
+var siteName = utag_data['context.site.siteName']
+var experimentIds = utag_data['marketingABTests'];
+var expIds = experimentIds.split(",");
+var counter=expIds.length;
+for(var expId=0; expId<counter; expId++) {
+  (function(getId) {
+   var xhttp = new XMLHttpRequest();
+	xhttp.open('GET', 'https://'+siteName+'/api/bucketing/v1/evaluateExperimentsAndLog?guid='+guid+'&tpid='+tpid+'&eapid='+eapid+'&id='+getId+'', true);
```

Comment:
any reason why you would want to modify the sitename? I think I have left a similar comment previously